packages/gestalt/src/Toast.js

```diff
@@ -66,6 +67,24 @@ export default function Toast({
   variant = 'default',
   _dangerouslySetPrimaryAction,
 }: Props): Node {
+  /*
+   *
+   * EXPERIMENT LOGIC START
+   *
+   */
+  const inToastExp = useInExperiment({
+    webExperimentName: 'web_gestalt_redesigned_toast',
+    mwebExperimentName: 'mweb_gestalt_redesigned_toast',
+  });
+
+  const EXP_SIZE_THUMBNAIL = 32;
```

Comment:
These should be defined outside of the function

Reply:
DONE